.github/workflows/ci.yml

```diff
@@ -18,9 +18,7 @@ jobs:
         python: ['3.10', '3.11', '3.12']
         django: ['3.2', '4.2']
         exclude:
-          - python: '3.11'
-            django: '3.2'
-          - python: '3.12'
+          - python: ['3.11', '3.12']
```

Comment:
this syntax does not work:

![image](https://github.com/maykinmedia/django-simple-certmanager/assets/5518550/cf42b7f3-b87e-4207-867c-b1744f80294e)

And it results in Django 5.0 being actually installed, so it's super misleading too:

```
py: freeze> python -m pip freeze --all
py: asgiref==3.7.2,black==24.2.0,cachetools==5.3.3,certifi==2024.2.2,cffi==1.16.0,chardet==5.2.0,click==8.1.7,colorama==0.4.6,coverage==7.4.3,cryptography==42.0.5,cssselect==1.2.0,distlib==0.3.8,Django==5.0.2,django-privates==2.0.0.post1,django-sendfile2==0.7.1,django-simple-certmanager==1.4.1,django-stubs==4.2.7,django-stubs-ext==4.2.7,filelock==3.13.1,flake8==7.0.0,freezegun==1.4.0,iniconfig==2.0.0,isort==5.13.2,lxml==5.1.0,mccabe==0.7.0,mypy==1.7.1,mypy-extensions==1.0.0,packaging==23.2,pathspec==0.12.1,pip==24.0,platformdirs==4.2.0,pluggy==1.4.0,pycodestyle==2.11.1,pycparser==2.21,pyflakes==3.2.0,pyproject-api==1.6.1,pyquery==2.0.0,pytest==8.0.2,pytest-cov==4.1.0,pytest-django==4.8.0,python-dateutil==2.9.0,six==1.16.0,sqlparse==0.4.4,tox==4.13.0,types-pytz==2024.1.0.20240203,types-PyYAML==6.0.12.12,typing_extensions==4.10.0,virtualenv==20.25.1
```

Reply:
:(

Reply:
Really wondering why they don't support it